dbg.py

```diff
@@ -57,17 +57,19 @@ def free(self, addr):
 		req.address = addr
 		return self._send_simple_request(req)
 
-	def mem(self, addr, value=None):
+	def mem(self, addr, size=0, data=None):
```

Comment:
split into mem_read and mem_write, in the future there might also be io_read and io_write